src/code/z_view.c

```diff
@@ -310,16 +310,17 @@ s32 View_ApplyPerspective(View* view) {
     height = view->viewport.bottomY - view->viewport.topY;
     aspect = (f32)width / (f32)height;
 
-    if (HREG(80) == 11) {
-        if (HREG(94) != 11) {
-            HREG(94) = 11;
-            HREG(83) = 60;
-            HREG(84) = 13333;
-            HREG(85) = 10;
-            HREG(86) = 12800;
-            HREG(87) = 100;
+    if (R_HREG_MODE == HREG_MODE_PERSPECTIVE) {
+        if (R_PERSPECTIVE_INIT != HREG_MODE_PERSPECTIVE) {
+            R_PERSPECTIVE_INIT = HREG_MODE_PERSPECTIVE;
+            R_PERSPECTIVE_FOVY = 60;
+            R_PERSPECTIVE_ASPECT = 13333;
```

Comment:
```suggestion
            R_PERSPECTIVE_ASPECT = (10000 * 4) / 3;
```
Does something like this match? I think it's nice to see how this number comes about